src/CommonComponents/Popup/index.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import popuplogo from '../../assets/popup-logo.png';
+import { PopUPCard, PopUpPox, Logo, LogoImage, BodyPage ,PopUpDiv,LeftButtonPopUp,RightButtonPopUp,TitleP } from './index.style';
+import popUpdata from '../../Data/popupdata'
+class PopUp extends React.Component {
+  state = {
+    title: '',
+    descraption: '',
+    leftButton: '',
+    rightButton: ''
+  };
+  render() {
```

Comment:
I think this component will not be able to used in different places, so please change it to stateless component and pass props to help you to use in different places